### PR TITLE
add manual test for PaymentResponse.payerName attr

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 06:26:47
Finished: 2017-08-21 06:36:09



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17590)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266686344)

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 06:26:47
Finished: 2017-08-21 06:36:09



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17590)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266686344)

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 06:26:47
Finished: 2017-08-21 06:36:09



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17590)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266686344)

File: payment-request/payment-response/payerName-attribute-manual.https.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <li>
+    <button onclick="runManualTest(this, { requestPayerName: undefined }, { payerName: null })">
+      payerName attribute is null when requestPayerName is undefined.
+    </button>
+  </li>
+  <li>
+    <button onclick="runManualTest(this, { requestPayerName: false }, { payerName: null })">
+      payerName attribute is null when requestPayerName is false.
+    </button>
+  </li>
```
Comment (edenchuang) [inline payment-request/payment-response/payerName-attribute-manual.https.html]: Maybe a test for null PaymentOption for test the default browser settings.
button onclick="runManualTest(this, null, { payerName: null })"

**Review by edenchuang — APPROVED** [review]
LGTM

File: payment-request/payment-response/payerName-attribute-manual.https.html (hunk 1 above)
Comment (marcoscaceres) [inline payment-request/payment-response/payerName-attribute-manual.https.html]: I think "runManualTest(this, null, { payerName: null })" will get converted to the string "null", which will be truthy.

Comment (marcoscaceres) [inline payment-request/payment-response/payerName-attribute-manual.https.html]: Added test above

**Review by edenchuang — APPROVED** [review]

**Pull request merged**